spinalcordtoolbox/scripts/sct_create_mask.py

```diff
@@ -242,23 +242,13 @@ def create_mask(param):
     for iz in range(0, nz, 1):
         if iz in z_centerline_not_null:
             cx[iz], cy[iz] = ndimage.measurements.center_of_mass(np.array(data_centerline[:, :, iz]))
-    # create 2d masks
-    file_mask = 'data_mask'
+
+    # create 2d masks at each axial slice
+    im_out = zeros_like(im_data)
```

Comment:
The previous error in CI was due to a bug in the `zeros_like` function. 

In this instance, `im_data.data` was an np.memmap. So, `zeros_like` would copy the image (including the np.memmap), which means the zero array would ovewrite the old `im_data`. 

This has been fixed in 7722b9b.

Reply:
Fixed in 07d9b15.